lntest/itest/lnd_test_list_on_test.go

```diff
@@ -274,4 +274,8 @@ var allTestCases = []*testCase{
 		name: "open channel fee policy",
 		test: testOpenChannelUpdateFeePolicy,
 	},
+	{
+		name: "neutrino",
```

Comment:
maybe call this "neutrino rpc test"? Calling it `neutrino` is too generic...

Reply:
Thanks for the guidance. Will change the commit messages asap.